chaindataloader/binary_test.go

```diff
@@ -0,0 +1,180 @@
+package chaindataloader
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightninglabs/neutrino/headerfs"
+)
+
+const (
+	testDataDir = "testdata/"
+)
+
+// TestNewBinarySideLoad tests that the reader returns the appropriate data
+// required for fetching headers when initialized.
+func TestNewBinarySideLoad(t *testing.T) {
+	type testSideLoadConfig struct {
+		name        string
+		path        string
+		endHeight   int64
+		startHeight int64
+		network     wire.BitcoinNet
+		tipHeight   int64
+		err         bool
+		tipErr      bool
+	}
+
+	testCases := []testSideLoadConfig{
+
+		{
+			name: "invalid network in header file",
+			path: "start_2_end_10_encoded_8_invalidNetwork_headers.bin",
+			err:  true,
+		},
+
```

Comment:
Style nit: can eliminate the new line between test cases if they don't have a top level comment. Also same rule here is 80 character columns as elsewhere.

Reply:
Do you think there is a way to include these rules as part of the lint?

Reply:
Yeah we do it for lnd which has a more advanced linter, so we can port it over here.